R/new-blog-post.R

```diff
@@ -0,0 +1,82 @@
+#' Create a new blog post
+#'
+#' Creates (and potentially opens) the `index.qmd` file for a new blog post.
+#'
+#' @param title A character string for the title of the post. It is converted
+#' to title case via [tools::toTitleCase()].
+#' @param dest A character string (or NULL) for the path within `posts`. By
+#' default, the title is adapted as the directory name.
+#' @param open A logical: have the default editor open a window to edit the
+#' `index.qmd` file?
+#' @param call A call object for reporting errors.
+#' @param ... A named list of values to be added to the yaml header, such as
+#' `description`, `author`, `categories`, etc.
+#' @return The path to the index file.
+#' @export
+#' @examples
+#' \dontrun{
```

Comment:
I think that this is the best way to protect the example.

Reply:
We've used that a lot with no real problems from cran. What condition should I use to skip (being tested on CRAN)?

Reply:
> We've used that a lot with no real problems from cran.

Then all good ! Let's keep it, and I handle CRAN if this is flagged for some reason